sources/web/datalab/polymer/components/datalab-files/datalab-files.ts

```diff
@@ -104,12 +119,20 @@ class FilesElement extends Polymer.Element {
                                     this._handleDoubleClicked.bind(this));
     }
 
-    this.$.files.columns = ['Name', 'Status'];
+    // For a small file/directory picker, we don't need to show the status.
+    this.$.files.columns = this.small ? ['Name'] : ['Name', 'Status'];
 
     // Refresh the file list periodically.
     // TODO: [yebrahim] Start periodic refresh when the window is in focus, and
     // the files page is open, and stop it on blur to minimize unnecessary traffic
-    setInterval(this._fetchFileList.bind(this), this._fileListRefreshInterval);
+    this._fileListRefreshIntervalHandle = setInterval(this._fetchFileList.bind(this),
```

Comment:
Prefer breaking lines at a higher semantic level; in this case, just after the "=".